_posts/en/newsletters/2020-09-02-newsletter.md

```diff
@@ -0,0 +1,252 @@
+  both parties, there will be a difference in the signatures (called
+  *witnesses*) each of them uses to unilaterally close the channel.
+  The difference in the signatures will give the party who didn't close
+  the channel a chance to create and broadcast a penalty transaction if
+  the channel was closed in an old state.  Otherwise, if the channel was
+  closed in its latest state, both parties get their money, just like in
+  the current LN.
+
+  The proposal also suggests using basically the same scheme to
+  transition from Hash Time Lock Contracts (HTLCs) to Point Time Lock
+  Contracts ([PTLCs][news92 ptlcs]), which is already a goal several LN contributors
+  have voiced support for in the past.
+
+  Advantages of this approach are that it might require less private
+  data storage (though this is [debated][zmn reply]), it makes unilateral closes
+  a bit smaller onchain, and it describes a path to using PTLCs that
+  will provide LN users with additional privacy against routing
+  node surveilance as well as bring the protocol a little closer to what it may
+  look like when it can take advantage of the benefits of [schnorr
+  signatures][topic schnorr signatures].
+
+- **Default signet discussion:** Michael Folkson [posted][default signet post] to the
+  Bitcoin-Dev mailing list a question about whether there should be a
+  default [signet][topic signet] similar to Bitcoin's existing default testnet.
+  Signets are test networks where valid blocks must be
+  signed by one or more trusted keys; this allows them to be more
+  predictable for testing than fully decentralized test networks that
+  are often subject to vandalism.  [BIP325][] describes the signet
+  protocol, which anyone can use to create their own signet, but there
+  is a [desire][bitcoin core default signet] to include a default signet into
+  Bitcoin Core (and possibly other software) that
+  users will be able to enable with a single configuration option.
+  Folkson's first question is whether anyone has any objection to this.
+
+    If there is to be a default signet, Folkson posted several
+    additional questions about how it should be administered.  For
+    example, who should control its signing keys?  How many keys should
+    there be?  Under what circumstances should the default signet be
+    reset?  When should proposed consensus changes to mainnet Bitcoin be
+    implemented by the trusted signet miners/signers?
+
+    Anyone interested in helping answer these questions is encouraged to
+    respond.
+
+- **Standardizing temporarily trusted LN channels:** Roei Erez
+  [posted][temporarily trusted channels] to the Lightning-Dev mailing list a proposal to standardize
+  the way LN implementations deal with new channels when the
+  participants are willing to trust an unconfirmed deposit transaction.
+  For example, Alice pays Bob's established business to fund a
+  channel to her.  In that case, Alice may be willing to trust Bob to not
```

Comment:
Is this describing `turbo channels`? If yes, should the term be referenced?

Reply:
I think that's the name BitRefill uses, but I don't think anyone else providing that service uses that term.

Reply:
afaik, @kixunil published the concept under this name...?

Reply:
I published the concept without any name. Then @btcontract [suggested] calling it "turbo"(https://github.com/lightningnetwork/lightning-rfc/issues/565#issuecomment-460601680) and Bitrefill decided to use that term in their campaigns which popularized the idea and the name. Since then I've heard and used "Turbo" in several cases outside Bitrefill-specific services. The [suggested technical term](https://github.com/lightningnetwork/lightning-rfc/issues/565#issuecomment-462119239) is `zeroconf_spendable_push`, which Anton agreed on.

I'm not aware of any other *popular* name for this feature, so I'd like to see these terms being used to avoid confusion. I'd recommend at least putting "(AKA Turbo)" somewhere in the text to make it more clear.